src/octoprint/plugins/action_command_notification/__init__.py

```diff
@@ -56,11 +58,13 @@ def get_assets(self):
     def on_event(self, event, payload):
         if event == Events.DISCONNECTED:
             self._clear_notifications()
+        elif event == Events.SETTINGS_UPDATED or event == Events.STARTUP:
+            self._configure_filters()
```

Comment:
I wouldn't do this via event handling (runs concurrently to the rest of the platform), but rather ensure it really is done before further actions by hooking into `initialized` and `on_settings_save`.